src/main/java/com/faforever/client/chat/ChatUserItemController.java

```diff
@@ -529,6 +476,35 @@ public void updateAvatarTooltip() {
     });
   }
 
+  private void updateGameStatus() {
+    if (chatUser == null) {
+      return;
+    }
+    Optional<Player> playerOptional = chatUser.getPlayer();
```

Comment:
The whole block could use Optional's ifPresentOrElse

Reply:
making the entire block less readable, yes